op-supervisor/supervisor/backend/depset/static.go

```diff
@@ -14,6 +14,13 @@ import (
 )
 
 var errDuplicateChainIndex = errors.New("duplicate chain index")
+var errUsingReservedChainIndex = errors.New("using reserved chain index")
+
+// NotFoundChainIndex is a reserved chain index signifying that a network is not in the dependency set.
+// We are persisting these invalid, but not malformed, executing messages in our internal database, and
+// we need a mapping between all ChainIDs <> ChainIndexs. However ChainID is 32 bytes, and ChainIndex
+// is 4 bytes, so we need a reserved chain index for unknown chains.
+var NotFoundChainIndex = types.ChainIndex(3735928559) // 0xDEADBEEF
```

Comment:
```suggestion
const NotFoundChainIndex = types.ChainIndex(0xFFFF_FFFF)
```

Let's just use the maximum index value, it's more predictable and readable.
And let's make it a constant; this shouldn't change.